etn-ccis/blui-angular-workflows
# Add github Blui CI

Fixes BLUI-5563.


#### Changes proposed in this Pull Request:
- add github CI workflow
- remove circleCI config
- stop building in circleCI
- upgrade cypress


#### Screenshots / Screen Recording (if applicable)
-



#### To Test:
- you can make a small commit and revert it and see jobs run.
- [branch codecov report](https://app.codecov.io/github/etn-ccis/blui-angular-workflows/tree/feature%2Fblui-5563-github-ci-workflow/login-workflow)
- workflow jobs https://github.com/etn-ccis/blui-angular-workflows/actions/runs/8527845928
- The report job is reporting different than the actual and have not figured out why coverage shows as less but in codecov dashboard the master branch & feature branch total coverage is the same. Don't know.... 


#### Any specific feedback you are looking for?
-
